doc/README.md

```diff
@@ -510,15 +491,14 @@ Parse a .bmap file
 
 <a name="BlockMap.stringify"></a>
 
-### BlockMap.stringify(blockMap, [options]) ⇒ <code>String</code>
+### BlockMap.stringify(blockMap) ⇒ <code>String</code>
 Stringify a block map into .bmap format
 
 **Kind**: static method of <code>[BlockMap](#BlockMap)</code>  
 **Returns**: <code>String</code> - xml  
 **Params**
 
 - blockMap <code>[BlockMap](#BlockMap)</code> - – BlockMap instance
```

Comment:
Because it's an "–" instead of a "-" in the code... I fatfingered onto <kbd>Alt</kbd> -.-'
Fixed now.